internal/controller/pipelinerollout/pipelinerollout_ppnd.go

```diff
@@ -101,7 +110,7 @@ func (r *PipelineRolloutReconciler) processExistingPipelineWithPPND(ctx context.
 	}
 
 	// are we done with PPND?
-	doneWithPPND := !shouldBePaused
+	doneWithPPND := resumed
```

Comment:
if pipeline is resumed (no longer in Paused or Pausing), then we can return true from this function, which will enable us to no longer be in ppnd mode, and then if the "numaflow.numaproj.io/resume-strategy"=="slow", we will set it back to that then.